index.js

```diff
@@ -6,13 +6,15 @@ var balanced = require("balanced-match")
 var colorFn = require("css-color-function")
 var helpers = require("postcss-message-helpers")
 
+var colorFnRegExp = /(^|[^\w\-])color\(/;
+
 /**
  * PostCSS plugin to transform color()
  */
 module.exports = postcss.plugin("postcss-color-function", function() {
   return function(style) {
     style.eachDecl(function transformDecl(decl) {
-      if (!decl.value || decl.value.indexOf("color(") === -1) {
+      if (!decl.value || !colorFnRegExp.test(decl.value)) {
```

Comment:
I test `RegExp#test` and `String#indexOf` on our case. https://gist.github.com/demiazz/b82fd832327367ceba04

`RegExp#test` faster, then `String#indexOf`.